### ISSUE TITLE
Rename Acknowledgement (for Member submission requests)

### ISSUE THREAD

**Issue by frivoal** [w3c/process#714]
I accidentally closed https://github.com/w3c/w3process/pull/640 and cannot reopen because reopening pull requests isn't a thing in GitHub, so opening this issue to follow up.

The initial problem statement was:
> Acknowledgement sounds like "yes, we got your request", while it intends to mean "yes, we agree to publish your submission". Using the "accept" to mean that seems less confusing.

We then rejected "accept" and tried "receive" for a while, but that didn't satisfy everyone either. So the latest exploration suggests that following potential alternatives:

* Verify / Verification
* Validate / Validation
* Confirm / Confirmation
* Recognize / Recognition
* Admit / Admission
* Permit / Permission
* Authorize / Authorization
* Ratify / Ratification
* Sanction / Sanctioning
* Permit / Permission
* Grant / Granted
* Clear / Clearance
* Greenlight / Greenlighting
* OK / Okayed
* Affirm / Affirmation

**Comment by nigelmegitt** [comment]
I'd go with "Admit / Admission" out of that list, though "recognize" might be made to work too.

Several of the others suffer the same issue that @swickr identified with "accept", that they could be taken to signify approval somehow. For that reason I'd exclude validate, authorize, ratify, sanction, permit, clear, greenlight, OK and possibly affirm.

Some of the others just don't seem to carry the right sense in terms of the process that is being described. Permit, grant and affirm fall into that category. 

Verify is interesting because it suggests that the only criteria is technical fulfilment of some well-formedness test - I'm actually not sure if that is the only criteria, or if some kind of review of the content is implied too.

**Comment by dwsinger** [comment]
I know this is controversial, but have we considered re-phrasing simply to say what we mean? That is, we either agree to publish it, or we don't (and if we agree to publish it, that does not imply any other endorsement, which is already stated).

So can we change to "agree to publish" and then for the most part we can use "published" and refer back to that agreement?

**Comment by frivoal** [comment]
The Process [specializes the word "publish" to mean "put something on TR"](https://www.w3.org/2021/Process-20211102/#publishing), and that's now what we mean here. Not sure the process is 100% successful at keeping to that meaning, but it seems to be trying to.

We could still do what you suggest, but we'd have to use some work-around phrasing, like "agree to make available to the public", or something like that.

(That said, the more I think about it, the more I'm tempted by https://github.com/w3c/w3process/issues/421)

**Comment by fantasai** [comment]
@nigelmegitt "Admit / Admission" works for me, and I'd be fine with "Recognize / Recognition" as well. Ping @swickr @koalie for thoughts.

**Comment by nigelmegitt** [comment]
Thanks @koalie , can you explain why you prefer "Recognize / Recognition" over "Admit / Admission"?

**Comment by koalie** [comment]
> Thanks @koalie , can you explain why you prefer "Recognize / Recognition" over "Admit / Admission"?

Yes, sorry. I find the former to be more neutral in meaning and expectations it sets. The latter is more evocative of endorsement than the former.

**Comment by koalie** [comment]
And now that I've considered them too much, they carry no meaning anymore (to the extent that my preference which was strong earlier today is no longer strong)! 
If dictionaries can back up our choice, that makes the choice stronger. @fantasai was ok with "Admit / Admission" first and "Recognize / Recognition" second, just like you @nigelmegitt

**Comment by chrisn** [comment]
I tend to agree with @dwsinger's comment - sometimes reaching for the thesaurus is an indication that a rephrasing is needed.
